sass/sass.bzl

```diff
@@ -103,6 +105,7 @@ def _run_sass(ctx, input, css_output, map_output = None):
         arguments = [args],
         outputs = [css_output, map_output] if map_output else [css_output],
         use_default_shell_env = True,
+        execution_requirements = {"supports-workers": "1"},
```

Comment:
Adding this line itself won't break anything, it is only effective if users have --strategy=SassCompiler=worker in the command or blazerc.

With that being said, this worker impl probably won't work inside google since it uses dart_binary.

Reply:
There may be some uses of `--strategy=SassCompiler=worker` lingering within Google from the SassC days. We should make sure this line gets deleted when it gets synced in, so that the error message users who pass this flag see will be clearer.

Reply:
Absolutely. I will be happy to change the copybara configurations in google3.